alicloud/resource_alicloud_oss_bucket.go

```diff
@@ -414,6 +419,26 @@ func resourceAlicloudOssBucketRead(d *schema.ResourceData, meta interface{}) err
 		}
 	}
 
+	// Read Policy
+	raw, err = client.WithOssClient(func(ossClient *oss.Client) (interface{}, error) {
+		params := map[string]interface{}{}
+		params["policy"] = nil
+		return ossClient.Conn.Do("GET", d.Id(), "", params, nil, nil, 0, nil)
+	})
+
+	rawResp := raw.(*oss.Response)
+	defer rawResp.Body.Close()
+
+	if err == nil {
```

Comment:
If there is an error, it should be throw.

Reply:
done